esp/logging/loggingmanager/loggingmanager.cpp

```diff
@@ -219,59 +219,44 @@ bool CLoggingManager::updateLog(IEspContext* espContext, IEspUpdateLogRequestWra
     return bRet;
 }
 
-static bool checkEnabledLogEntry(IPropertyTree *scriptValues, const char *profile, const char *group, unsigned &groupEntryCount, const char *logtype, unsigned &typeEntryCount)
+static bool checkEnabledLogVariant(IPropertyTree *scriptValues, const char *profile, const char *tracename, const char *group, const char *logtype)
 {
-    if ((!group || *group) && (!logtype || !*logtype)) //get this special case out of the way
+    bool checkProfile = !isEmptyString(profile);
+    bool checkType = !isEmptyString(logtype);
+
+    if (checkProfile && (isEmptyString(group) || !strieq(profile, group)))
         return false;
```

Comment:
With a log message for entry disabled by type, it seems incomplete to not log an entry disabled by profile. The log won't be reliable for debugging if only some disabled entries are included.

Reply:
true, I'll add one.